source/blog/napari-xarray.md

```diff
@@ -0,0 +1,168 @@
+
+But something being hard is not a reason not to try to do it. In that spirit both the napari and Xarray developers are committed to improving the integration of napari and Xarray. This blog post is the kickoff of that collaboration
+
+## How
+
+### SciPy 2025
+
+Thanks to support from the Chan Zuckerberg Institute the napari, Xarray, and [CellProfiler](https://cellprofiler.org/) team members were able to attend SciPy 2025. At the sprints members of these three teams—Ian Hunt-Isaak, Tim Monko, Nodar Gogoberidze, Beth Cimini, Peter Sobolewski and Carol Willing—collaborated to develop a plan for enhancing the integration of Xarray with napari. The rest of this blog post is the initial roadmap we came up with for better integration of Xarray and napari.
+
+### Get Involved
+
+This is a community effort, so your contributions and thoughts are very welcome! To get involved in shaping this vision of the future, please join the napari [Zulip](https://napari.zulipchat.com/) and introduce yourself on this [introductions issue](https://github.com/napari/napari-xarray/issues/8) in the napari Xarray repository.
+
+### **🗺️ Roadmap**
+
+To fully realize the potential of an integration between Xarray and napari, some deep changes in napari may be required. Therefore, we have developed a phased plan that progresses from simple to more complex enhancements, culminating in solutions that require more fundamental changes to how napari handles data.
+
+#### 1: Basic Metadata Mapping (Proof of Concept)
+
+This first step is the easiest, as it requires no changes to napari's core and can be implemented in a simple script or plugin.
+
+**Goal:** Ingest Xarray data into napari's layer metadata to provide immediate context to the user.
+
+**Key Functionality:**
+
+- Auto-label sliders with Xarray dimension names (e.g., `'Z'`, `'Time'`)
+- Assign layer names from Xarray's `.name` attribute
+- Automatically transform:
+  - [`DataArray`](https://docs.xarray.dev/en/latest/user-guide/data-structures.html#dataarray) -> Layer
+  - [`Dataset`](https://docs.xarray.dev/en/latest/user-guide/data-structures.html#dataset) -> Multiple Layers
+  - [`DataTree`](https://docs.xarray.dev/en/latest/user-guide/data-structures.html#datatree) -> Multiple Layers + Pyramidal Viewing (if applicable)
+
+**Implementation:** Script or plugin level, no napari core changes required. This will be an opportunity 
+
+**napari enhancement** napari has laid the groundwork for success here with the ability to include `axis_labels` and `units` on layers in [this pr](https://github.com/napari/napari/pull/6979). The remaining work is to close the gap by adding connections from the napari layer list to the napari dims model. This improvement will help all napari users, not just those using xarray.
```

Comment:
>   Not quite done with a full read through but I've left a few comments for now. Most important is that we already have axis_labels on layers and we should hook that up to the dims model!

in the spirit of finishing this, and brevity to keep the reader engaged I added this section that gets at what I htink @jni was suggesting without getting too caught up in detailed planning here.

Reply:
I like the framing of this. Let's go for it.